pkgs/by-name/pr/protege/package.nix

```diff
@@ -0,0 +1,93 @@
+{ lib
+, fetchFromGitHub
+, copyDesktopItems
+, iconConvTools
+, makeDesktopItem
+, makeWrapper
+, jdk11
+, maven
+}:
+
+let
+  mvn = maven.override { jdk = jdk11; };
+
+  pname = "protege";
+  version = "5.6.4";
+
+  meta = {
+    homepage = "https://protege.stanford.edu/";
+    downloadPage = "https://protege.stanford.edu/software.php#desktop-protege";
+    description = "A free and open-source OWL 2 ontology editor";
```

Comment:
It might be a bit more readable to define only this `description` attribute in the `let .. in` part, so most of the meta will be found where it is always found - at the bottom.